### PR TITLE
Logseq file path encode, Auto source helpers rename

### PR DESCRIPTION
**Pull request by yoyurec**
Encode special chars in file path

### CODE REVIEW

File: doc/config.py [hunk 1]
```diff
@@ -14,6 +14,10 @@
 # - structured formats like JSON and CSV
 from promnesia.sources import auto
 
+# Logseq protocol support for auto indexer
+from promnesia.sources import logseq
```
Comment (karlicoss) [inline doc/config.py]: don't think it's actually needed here? it only contains helper functions for `auto` module, no need to import it in the config

Comment (yoyurec) [reply doc/config.py]: right. thought i've added config but forgot inport for it )) but it's not a source it's a replacer

Comment (yoyurec) [reply doc/config.py]: little confusing. i think as it's not THE SOURCE, but just replacer ("helper") - it should be renamed to `auto_logseq` & `auto_obsidian` or even put them to subfolder `auto`

Comment (karlicoss) [reply doc/config.py]: yep, good idea!

Comment (yoyurec) [reply doc/config.py]: fixed!

**Pull request merged**